Codeinwp/neve
# fix: dark-mode logo shown in light-mode on first visit [#4087]

### Summary
Fix the dark-mode logo shown in light mode on first visit

### Will affect the visual aspect of the product
NO

### Screenshots 

### Test instructions
1. Add a palette switcher in the header
2. Add different logos for Light palette and Dark palette
3. Check the logo in frontend in incognito mode
4. This is actually a regression. Please also check https://github.com/Codeinwp/neve/issues/4025  does not happen anymore.

## Check before Pull Request is ready:



Closes #4087.
